routes/course.js

```diff
@@ -67,41 +67,32 @@ router.get("/api/coursesByTerm/:plan_id", (req, res) => {
 //distribution field is also updated
 router.post("/api/courses", async (req, res) => {
   const course = req.body;
-  //console.log("course is ", course);
   await plans
     .findById(course.plan_id)
     .then((plan) => {
-      //console.log(plan);
       course.distribution_ids.forEach((id) => {
-        if (!plan.distribution_ids.includes(id)) {
+        if (!plan.distribution_ids.includes(id)) 
```

Comment:
Here we tested two scenarios: 1) omit distribution_ids from the api/course post request 2) include distribution_ids in the request but give it faulty values. 

Scenario 1 gave 500 (line 81) and scenario 2 gave 400 (line 75). It also crashed the server with a `[ERR_HTTP_HEADERS_SENT](https://stackoverflow.com/questions/52122272/err-http-headers-sent-cannot-set-headers-after-they-are-sent-to-the-client)`. The errorHandler sets a header first and line 131 returnData tries to set a new header (200) which gives us the error.